wandb/sdk/wandb_settings.py

```diff
@@ -505,6 +505,11 @@ def _default_props(self) -> Dict[str, Dict[str, Any]]:
         """
         return dict(
             _disable_meta={"preprocessor": _str_as_bool},
+            _disable_service={
+                "value": False,
+                "preprocessor": _str_as_bool,
+                "is_policy": True,
```

Comment:
Is this necessary?

Reply:
(I mean is_policy)

Reply:
I want it to retain the backward behavior. I don't want users to pass it as a setting to `wandb.run` and overwrite env variable setting